tests/v2/test_waku_rln_relay.nim

```diff
@@ -2,99 +2,141 @@ import
+# the address of Ethereum client (ganache-cli for now)
+const ethClient = "ws://localhost:8541/"
+
+const poseidonHasherCode = readFile("tests/v2/poseidonHasher.txt")
```

Comment:
Can we add a link to how this contract code was found? Where is the canonical source of it? A few lines of comments here would be good